playbooks/nfd_operator_status_check.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
I don't see any `toolbox` script nor [`ci_entrypoint_nfd-operator.sh`](https://github.com/openshift-psap/ci-artifacts/blob/master/build/root/usr/local/bin/ci_entrypoint_nfd-operator.sh) call to this playbook (the entrypoint should call the toolbox, and the toolbox launches the playbook)

before the review can be finalized we'll need to have the code running as a PR test (with `/test nfd-operator-e2e-master`)